src/pages/collections/[collectionId]/review.tsx

```diff
@@ -207,18 +207,18 @@ const CollectionItemGridItem = ({ data: collectionItem }: CollectionItemGridItem
   return (
     <FeedCard>
       <Box className={sharedClasses.root} onClick={() => toggleSelected(collectionItem.id)}>
-        <Card.Section>
-          <Checkbox
-            checked={selected}
-            onChange={() => toggleSelected(collectionItem.id)}
-            size="lg"
-            sx={{
-              position: 'absolute',
-              top: 5,
-              right: 5,
-              zIndex: 9,
-            }}
-          />
+        <Checkbox
+          checked={selected}
+          onChange={() => toggleSelected(collectionItem.id)}
```

Comment:
I believe we can make this checkbox readOnly so we don't handle toggle state in both the checkbox itself and the card wrapper
```suggestion
          readOnly
```